packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -470,10 +471,19 @@ export class RunningSummarizer implements IDisposable {
 			let overrideDelaySeconds: number | undefined;
 			let summaryAttempts = 0;
 			let summaryAttemptsPerPhase = 0;
+			// Reducing the default number of attempts to defaultNumberofSummarizationAttempts.
+			const totalAttempts =
+				loggerToMonitoringContext(this.logger).config.getNumber(
+					"Fluid.Summarizer.Attempts",
+				) ?? defaultNumberSummarizationAttempts;
+			assert(
```

Comment:
Instead of assert a UsageError might be better since its due to incorrect usage of the feature flag.

Reply:
throwing an UsageError instead.